File: AssessmentApplication.Web/ngApps/SalesOrder/Models/Person.js
function Person(businessEntitiyId, title, firstName, middleName, lastName, suffix) {
	this.businessEntityId = businessEntitiyId;
	this.title = title;
	this.firstName = firstName;
	this.middleName = middleName;
	this.lastName = lastName;
	this.suffix = suffix;
	this.fullName = [title, firstName, middleName, lastName, suffix]
		.filter(function(val) {
			if (val) {
				return val;
			}
		})
		.join(" ");
}

File: AssessmentApplication.DataContracts/Person.cs
using System;
using Newtonsoft.Json;

namespace AssessmentApplication.DataContracts
{
    [JsonObject(MemberSerialization.OptIn)]
    [Serializable]
    public class Person
    {
        #region Public Properties

        [JsonProperty]
        public int BusinessEntityId { get; set; }

        [JsonProperty]
        public string FirstName { get; set; }

        [JsonProperty]
        public string LastName { get; set; }

        [JsonProperty]
        public string MiddleName { get; set; }

        [JsonProperty]
        public string Suffix { get; set; }

        [JsonProperty]
        public string Title { get; set; }

        #endregion Public Properties
    }
}
